### ISSUE TITLE
Show user notifications

### ISSUE THREAD

**Issue by fiatjaf** [websitesfortrello/trelew#6]
We have to integrate this somewhere somehow. If you have an idea of how is the approach way or want to work on this, please write here!

**Comment by fiatjaf** [comment]
codemill${5}

**Comment by fiatjaf** [comment]
codemill$5

**Comment by semanser** [comment]
What about using this module?
https://github.com/mikaelbr/node-notifier

**Comment by fiatjaf** [comment]
@semanser no, this is a command line application, there should be no GUI elements, it would defeat the purpose.
